third_party/nvfuser/csrc/graph_fuser.cpp

```diff
@@ -1084,7 +1084,24 @@ struct CudaGraphFuser {
         shape_of.emplace(n->output(), size);
         continue;
       }
+      if (n->kind() == aten::index_select) {
+        TORCH_INTERNAL_ASSERT(
+            shape_of.count(n->input(0)) > 0,
+            "buildShapeExpressions failed at accessing input shapes");
+        Node* dim_const = graph->createClone(n->input(1)->node(), map_inputs);
```

Comment:
nitpick, can we add a check `n->input(1)->node()` is a constant node?

Reply:
Done